workbench-core/authorization/src/__mocks__/permissionsPlugin.ts

```diff
@@ -16,7 +16,7 @@ export class MockPermissionsPlugin implements PermissionsPlugin {
   private _routesIgnored: RoutesIgnored = { '/login': { GET: true }, '/sample/*': { GET: true } };
 
   public async getPermissionsByUser(user: AuthenticatedUser): Promise<Permission[]> {
-    if (user.roles.includes('admin')) {
+    if (user.roles.includes('itadmin')) {
```

Comment:
do we care that this changed from `ITAdmin` to `itadmin` in these tests? I assume these are unit tests isolated from the integration tests.